aiohttp/web_urldispatcher.py

```diff
@@ -315,6 +315,12 @@ def register_route(self, route):
                 self._routes[name] = route
         self._urls.append(route)
 
+    def add_routes(self, methods, path, handler,
+                   *args, name=None, expect_handler=None):
```

Comment:
The method signature is ambiguous: when you call it with `name` parameter it should raise `ValueError` by https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/web_urldispatcher.py#L311

Also the method should return list of `Route` instances retrieved by `.add_route()` call.